py/visdom/server.py

```diff
@@ -1653,8 +1653,6 @@ def download_scripts(proxies=None, install_dir=None):
             'react-dom.min.js',
         '%sreact-modal@3.1.10/dist/react-modal.min.js' % b:
             'react-modal.min.js',
-        'https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_SVG':  # noqa
-            'mathjax-MathJax.js',
```

Comment:
Rather than removing this download entirely and committing the MathJax files to the `Visdom` repo, you should add extra download paths here to create and fill out the `static/js/MathJax` folder with the files you've found are required to get everything to work.

Reply:
Ok I will try it out. But may be I need to write another code for that. I will try using the available one. But if not then I ill write my own code there and it will not break other cod, for sure.

Reply:
I have added the code